modules/swagger-codegen/src/main/resources/objc/ApiClient-body.mustache

```diff
@@ -75,7 +75,9 @@ static void (^reachabilityChangeBlock)(int);
         [config.loggingFileHanlder writeData:[message dataUsingEncoding:NSUTF8StringEncoding]];
     }
 
-    NSLog(@"%@", message);
+    if ([[{{classPrefix}}Configuration sharedConfig] debug]){
```

Comment:
In other clients, the default output is the console. One can change it to a file instead but the debug flag must be switched on in order for the log to be written to the debug file.